CHANGELOG.md

```diff
@@ -28,6 +28,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Changed order of `additional_args` for Mysql dump runner, so additional options are added _after_ the existing args.
 - Added cli warnings for deprecated positional arguments and environmental variables.
 - Added Python 3.9 to official support description(pypi) and unit test targets.
+- Fixed The extra f typo in mssql , mysql and postgres i.e changed
+  ```py
+  self.logger.info(f"Running f{title} script #{i} \"{script[:50]}\"")
+  ```
+  to
+  ```py
+  self.logger.info(f"Running {title} script #{i} \"{script[:50]}\"")
+  ```
```

Comment:
1. Unfortunately this is in the wrong place - it should be under the `[Unreleased]` section because, well, we haven't released them! 
2. If you look at the other entries in the changelog, or other changelog examples, either from keepachangelog or other projects, adding a code-level description of what changed isn't what we want. A single sentence "Fixed a typo in mssql, mysql and postgres when running strategy before/after scripts" is the kind of thing I'm looking for. 

I mentioned 1 in my comment, I had thought 2 was covered by the style of keepachangelog standard - is there something we could add to the project to make this clearer/easier for new contributors?

Reply:
Actually, I'm so sorry I didn't see that I was making a change to a released version. It was due to my mistake. But no worries I will fix it... You just tell me are you fine with the branch name or I need to change that too

Reply:
Actually, I can't express what changes I made in English. It's not that I don't know English but I fail miserably while explaining code in English... but, now that you have given me what I should I write in the CHANGELOG.md I will copy paste it :sweat_smile:

Reply:
Everything else looks good 😇

I appreciate this might come off like I'm nitpicking everything, I welcome your contribution. I think a good thing to remember would be if you're not sure, ask! 

It's open source, it's a _together_ thing 🤓

Reply:
> Everything else looks good 😇
> 
> I appreciate this might come off like I'm nitpicking everything, I welcome your contribution. I think a good thing to remember would be if you're not sure, ask!
> 
> It's open source, it's a _together_ thing 🤓

Thanks, I really appreciate your support.